spaces/domains.json

```
@@ -452,5 +452,6 @@
   "vote.dios.finance": "diosfinance.eth",
   "gov.diversecapital.finance": "diversecapital.eth",
   "vote.feudalz.io": "feudalz.eth",
-  "vote.liquidrium.com": "liquidrium.eth"
+  "vote.liquidrium.com": "liquidrium.eth",
+  "vote.cask.fi":"caskprotocol.eth"
```

Please add a space between sub domain and ENS.
```suggestion
  "vote.cask.fi": "caskprotocol.eth"
```